### ISSUE TITLE
About the validation_path of Transformer training.

### ISSUE THREAD

**Issue by wbslye** [raywzy/ICT#30]
There is  a parameter--"validation_path"  described as "validation_image_path" in training Transformers,but I couldn't find a  parameter like "validation_mask_path" in "./Transformer/main.py".So does it mean that the validation set doesn't need its own mask set or it use the same mask set of training set or something else?Sorry for my poor English.

**Comment by raywzy** [comment]
Yes. Currently the validation shares the same mask set with the training. You could also pass some extra parameters to change this behavior.

**Comment by wbslye** [comment]
> Yes. Currently the validation shares the same mask set with the training. You could also pass some extra parameters to change this behavior.

Thanks for your reply.Have a good day! :D
